emhttp/plugins/dynamix/sheets/DashStats.css

```
@@ -1,78 +1,403 @@
+.view3,
+.view4,
+.unused {
+    display: none;
+}
+.port_view,
+.control {
+    z-index: 10000;
+}
+.flat {
+    height: 0;
+    lineheight: 0;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)